docs/openclaw-gateway-bridge-scheduling.md

```
@@ -0,0 +1,178 @@
+
+## Overriding the default cadence
+
+Instead of editing the shipped unit, drop a small override:
+
+```bash
+sudo systemctl edit openclaw-gateway-bridge.timer
+```
+
+In the editor, add:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)